### PR TITLE
Online lab environments

### LINKED ISSUE #1887: New "Online lab environments" page for the Install section above the advanced setup dropdown
### URL, if applicable

_No response_

### Describe the new content you are requesting.

We would like to create a new page in the install section letting users know of the various cloud development environment options available for them to use with Qiskit, as an alternative to working on their local machine. This content is being pulled from [this IBM Quantum blog post](https://www.ibm.com/quantum/blog/qiskit-notebook-environments), and is being surfaced in documentation for visibility. A new link will be provided in tutorials that links to this docs page, and supports the recent deprecation of runnable code cells in those tutorials. 

Proposed title (from working with Abby): Online notebook (or lab) environments

Rough content (forgive me for any major spelling issues, and we definitely need the install code double checked):

In addition to downloading and installing Qiskit to work on your local machine, there are several other options available for those that prefer to work in the cloud, in an online development environment. 

Several providers offer online Jupyter environments with Qiskit pre-installed for fast and convenient onboarding. These recommended providers include the [cloud service provider OVHcloud](https://fr.newsroom.ibm.com/OVHcloud-propose-le-kit-de-developpement-logiciel-Qiskit-dIBM-dans-le-cadre-de-son-portefeuille-de-notebooks-quantiques) and the [quantum software development cloud platform qBraid](https://www.qbraid.com/blog/qbraid-the-premier-destination-for-ibm-quantum-lab-users?utm_source=ibm&utm_medium=blog&utm_campaign=tech). 

Additional online Jupyter notebook environments like IBM Watson® Studio, Google Colab, and Microsoft Azure Machine Learning Studio can be used to work with Qiskit by following the instructions below. 

### Getting started with OVH Cloud AI Notebooks:
Users of OVHcloud Public Cloud and OVHcloud Startup can now leverage their OVHcloud quantum notebooks to access Qiskit and submit jobs to IBM Quantum hardware with their IBM Quantum or IBM Cloud® account. 
Note: OVHcloud AI notebooks are only available in select regions. 

(1) Set up OVHcloud free account
* Visit the [OVH Cloud landing page](https://www.ovhcloud.com/).
* Click on "Create an account" and fill in your details.
*
(2) Create an OVH Public Cloud project
* Log in and navigate to the Public Cloud page.
* Click on the "Create an AI Notebook" button.
*
(3) Create and run your Qiskit notebook
* Click on "+ Create a notebook".
* Name your notebook, select JupyterLab, choose Qiskit, set privacy settings, and select a datacenter location.  

### Getting started with qBraid Lab:
Qiskit users can leverage qBraid’s pre-configured Python environments using the instructions below. For full platform documentation, [click here.](https://docs.qbraid.com/home/introduction)  

(1) Set Up qBraid Free Account
* Visit the [qBraid Lab landing page](https://www.qbraid.com/).
* Click on "Create an account" and fill in your details.

(2) Create a Jupyter Lab Environment and Run Qiskit
* Log in and navigate to the Lab page.
* Click "Start Machine" in the free tier.  

## Additional notebook environments for Qiskit users
For platforms without Qiskit pre-configured, the setup instructions are largely the same. Users will need to create an account, start a new project (or “notebook” or “workspace,” depending on the platform), and pip install Qiskit.   

### Getting started with IBM Watson Studio
(1) Set Up Watson Studio Free Trial
* Visit the [Watson Studio landing page](https://dataplatform.cloud.ibm.com/registration/stepone?context=cpdaas&apps=data_science_experience%2Cwatson_machine_learning%2Ccos&utm_content=CPDWW).
* Create an IBM Cloud account and start a free trial.

(2) Create a Watson Studio Project
* Click on "New Project" and follow the prompts.
(3) Install Qiskit Packages and dependencies needed
* Open a new notebook and run:

`!pip install qiskit qiskit-ibm-runtime pylatexenc !yes | pip uninstall simplejson`

### Getting Started with Google Colab
(1) Set Up Google Account
* Visit the [Google Colab landing page](https://colab.research.google.com/signup).
* Sign in with your Google account or create a new one.
(2) Create a Notebook and Install Qiskit
* Click on "+ New Notebook" and run:

`%pip install qiskit qiskit-ibm-runtime`

### Getting Started with Microsoft Azure Machine Learning Studio
(1) Set Up Azure Account
* Visit the [Azure landing page](https://azure.microsoft.com/).
* Create a free trial account.
(2) Create Azure ML Studio Workspace and Install Qiskit
* Create a new workspace and a compute resource.
* Open a new notebook and run:

`%pip install qiskit qiskit-ibm-runtime`
  
And that is it! If you need help getting started with any of these online notebook environments, be sure to check the corresponding platform documentation, or consider dropping a question in [the Qiskit Slack channel](https://qiskit.slack.com/).

### If this new content request is accepted, do you want to write the content?

I can help the team by providing enough information to write the material

### PR DESCRIPTION
**Pull request by beckykd**
closes #1887

### CODE REVIEW

**Comment by beckykd** [comment]
@JulesMurphy I still have some behind-the-scenes stuff to do, but you can start reviewing the content.

File: docs/guides/online-lab-environments.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+</Admonition>
+
+Follow these steps to get started:
+
+1. Set up an OVHcloud account.
+
+    1. Visit the [OVH Cloud landing page.](https://us.ovhcloud.com/)
+    3. In the Public Cloud box, click **Learn more**.
+    4. In the **Public Cloud** section, click **Get Started**.
+    5. In the **I'm new to OVHcloud _xx_** box, enter your details and click **Create an account**.
```
Comment (JulesMurphy) [inline docs/guides/online-lab-environments.mdx]: what's up with the _xx_? Is it just a formatting thing @beckykd ?

File: docs/guides/online-lab-environments.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    7. After confirming your email address, finish creating the account.
+
+2. Create an OVH Public Cloud project.
+
+    1. Log in and navigate to the [Public Cloud page.](https://us.ovhcloud.com/manager/#/public-cloud/pci/projects/new?context=website)
+    2. Name your project and click **Create my project**.
+    3. ******************At this POINT YOU HAVE TO PAY.
```
Comment (JulesMurphy) [inline docs/guides/online-lab-environments.mdx]: I assume you also have some clarity you want to provide here @beckykd, but just flagging it regardless (:

File: docs/guides/online-lab-environments.mdx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+Several providers offer online Jupyter environments with Qiskit pre-installed for fast and convenient onboarding. These recommended providers include the cloud service provider OVHcloud and the quantum software development cloud platform qBraid.
+
+Additional online Jupyter notebook environments like IBM Watson&reg; Studio, Google Colab, and Microsoft Azure Machine Learning Studio can be used to work with Qiskit by following the instructions below.
+
+## Recommended notebook environments with Qiskit pre-installed
+
+### OVH Cloud AI Notebooks
+
+Users of OVHcloud Public Cloud and OVHcloud Startup can now leverage their OVHcloud quantum notebooks to access Qiskit and submit jobs to IBM Quantum hardware with their IBM Quantum or IBM Cloud&reg; account.
```
Comment (JulesMurphy) [inline docs/guides/online-lab-environments.mdx]: Do we want to be more clear here that first users will need to create one of those accounts to be able to run on hardware? Or maybe make sure we call it out in each of the steps? Your call, just want to make sure we drive it home somewhere.

File: docs/guides/online-lab-environments.mdx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+### Getting started with qBraid Lab
+
+Qiskit users can leverage qBraid’s pre-configured Python environments by following the instructions below. For more information, see the [qBraid Docs.](https://docs.qbraid.com/home/introduction)
+
+(1) Set Up qBraid Free Account
+
+Visit the qBraid Lab landing page.
+Click on "Create an account" and fill in your details.
+(2) Create a Jupyter Lab Environment and Run Qiskit
```
Comment (JulesMurphy) [inline docs/guides/online-lab-environments.mdx]: Looks like at least in preview mode there is an issue where (2) is not wrapping to a new line. Not sure if it is a limitation of this UI but would be fixed in actual docs staging 🤷

File: docs/guides/online-lab-environments.mdx [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+Log in and navigate to the Lab page.
+Click "Start Machine" in the free tier.
```
Comment (JulesMurphy) [inline docs/guides/online-lab-environments.mdx]: Do we want to add any additional context (like you have for OVH above) about how to specify Qiskit as the language of choice?

File: docs/guides/online-lab-environments.mdx [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+## Additional notebook environments for Qiskit users
+
+For platforms without Qiskit pre-configured, the setup instructions are largely the same. Users will need to create an account, start a new project (or “notebook” or “workspace,” depending on the platform), and pip install Qiskit.
```
Comment (JulesMurphy) [inline docs/guides/online-lab-environments.mdx]: "and start a new project (stuff). Then the only addition/different is that users will have to pip install Qiskit"

^^ I know my copy isn't great, it is just that reading that sentence as-is, it isn't 100% clear that the install step is the additional step"

File: docs/guides/online-lab-environments.mdx [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+### Getting Started with Microsoft Azure Machine Learning Studio
+
+(1) Set Up Azure Account
+
+Visit the Azure landing page.
+Create a free trial account.
+(2) Create Azure ML Studio Workspace and Install Qiskit
+Create a new workspace and a compute resource.
+Open a new notebook and run:
+%pip install qiskit qiskit-ibm-runtime
```
Comment (JulesMurphy) [inline docs/guides/online-lab-environments.mdx]: My only final comment overall is in the first step of each of the instructions, can we add a link to where users should go for each step? Like to the landing or account setup pages called out in the instructions? Just to help users find the right jumping-off point.

File: docs/guides/online-lab-environments.mdx [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+And that is it! If you need help getting started with any of these online notebook environments, be sure to check the corresponding platform documentation, or consider dropping a question in the Qiskit Slack channel.
```
Comment (JulesMurphy) [inline docs/guides/online-lab-environments.mdx]: Should we provide a link to the Qiskit slack channel?

File: docs/guides/online-lab-environments.mdx (hunk 1 above)
Comment (beckykd) [reply docs/guides/online-lab-environments.mdx]: > what's up with the _xx_? Is it just a formatting thing @beckykd ?

It's your country code

File: docs/guides/online-lab-environments.mdx (hunk 2 above)
Comment (beckykd) [reply docs/guides/online-lab-environments.mdx]: Yes - good catch

File: docs/guides/online-lab-environments.mdx (hunk 4 above)
Comment (beckykd) [reply docs/guides/online-lab-environments.mdx]: This is old too - not in the current file :(

File: docs/guides/online-lab-environments.mdx [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+Several providers offer online Jupyter environments with Qiskit preinstalled for fast and convenient onboarding. These recommended providers include the cloud service provider OVHcloud and the quantum software development cloud platform qBraid.
+
+Additional online Jupyter notebook environments like IBM Watson&reg; Studio, Google Colab, and Microsoft Azure Machine Learning Studio can be used to work with Qiskit by following the instructions below.
+
+## Recommended notebook environments with Qiskit preinstalled
+
+### OVH Cloud AI notebooks
+
+OVHcloud Public Cloud and OVHcloud Startup users can use their OVHcloud quantum notebooks to access Qiskit and submit jobs to IBM Quantum hardware with their IBM Quantum or IBM Cloud&reg; account. To learn more, visit the [OVHcloud Documentation and tutorials](https://help.ovhcloud.com/csm/world-documentation?id=kb_home) page.
```
Comment (abbycross) [inline docs/guides/online-lab-environments.mdx:19]:
```suggestion
OVHcloud Public Cloud and OVHcloud Startup users can use their OVHcloud quantum notebooks to access Qiskit and submit jobs to IBM&reg; QPUs with their IBM Quantum&trade; or IBM Cloud&reg; account. To learn more, visit the [OVHcloud Documentation and tutorials](https://help.ovhcloud.com/csm/world-documentation?id=kb_home) page.
```

File: docs/guides/online-lab-environments.mdx [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+1. [Set up an OVHcloud account.](https://help.ovhcloud.com/csm/en-account-create-ovhcloud-account?id=kb_article_view&sysparm_article=KB0043022)
+2. [Create an OVH Public Cloud project.](https://support.us.ovhcloud.com/hc/en-us/articles/360002245784-Creating-Your-First-Public-Cloud-Project)
+6. Follow the steps in [Launch your first AI Notebook,](https://help.ovhcloud.com/csm/en-public-cloud-ai-notebooks-definition?id=kb_article_view&sysparm_article=KB0048270) selecting **Qiskit** for the framework.
+
+### qBraid Lab
+
+Qiskit users can use qBraid’s preconfigured Python environments by following these instructions. For more information, see the [qBraid Docs.](https://docs.qbraid.com/home/introduction)
+
+1. [Set up an qBraid free account.](https://docs.qbraid.com/home/account)
+3. [Create a Jupyter Lab Environment.](https://docs.qbraid.com/lab/user-guide/getting-started)
```
Comment (abbycross) [inline docs/guides/online-lab-environments.mdx:37]:
```suggestion
3. [Create a JupyterLab environment.](https://docs.qbraid.com/lab/user-guide/getting-started)
```
I most usually see this as one word (and a lowercase e) but am not positive it's right in this context?...

File: docs/guides/online-lab-environments.mdx [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+3. Launch qBraid Lab, click the **ENVS** icon on the right, click **+ ADD** at the top of the new sidebar, search for Qiskit.  Click the version you want to use, then click **Install**.
```
Comment (abbycross) [inline docs/guides/online-lab-environments.mdx:38]:
```suggestion
3. Launch qBraid Lab, click the **ENVS** icon on the right, click **+ ADD** at the top of the new sidebar, and search for Qiskit.  Click the version you want to use, then click **Install**.
```

File: docs/guides/online-lab-environments.mdx [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+4. Create a Jupyter notebook by clicking **Python 3 [Qiskit]** in the Notebook section.
+4. You can install any additional Python packages (Such as Qiskit 1.2) in a notebook cell by running `%pip install [package_name].` For more information, see the [Pip (magic) commands](https://docs.qbraid.com/lab/user-guide/notebooks#pip-magic-commands) topic.
```
Comment (abbycross) [inline docs/guides/online-lab-environments.mdx:40]:
```suggestion
4. You can install any additional Python packages (Such as Qiskit v1.2) in a notebook cell by running `%pip install [package_name].` For more information, see the [Pip (magic) commands](https://docs.qbraid.com/lab/user-guide/notebooks#pip-magic-commands) topic.
```

File: docs/guides/online-lab-environments.mdx [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+3. [Create a Jupyter notebook in your workspace.](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-run-jupyter-notebooks?view=azureml-api-2)
+4. From the notebook, run this command to install Qiskit:
+   ```python
+    %pip install qiskit qiskit-ibm-runtime
+   ```
+
+## Helpful links
+
+<Admonition type="tip" title="Documentation resources">
+  -  [OVH Cloud documentation](https://help.ovhcloud.com/csm/en-documentation?id=kb_home)
```
Comment (abbycross) [inline docs/guides/online-lab-environments.mdx:77]:
```suggestion
  -  [OVHcloud documentation](https://help.ovhcloud.com/csm/en-documentation?id=kb_home)
```

**Review by abbycross — APPROVED** [review]
teensy things - all set otherwise!

File: docs/guides/online-lab-environments.mdx (hunk 10 above)
Comment (beckykd) [inline docs/guides/online-lab-environments.mdx:37]:
```suggestion
3. [Create a Lab environment.](https://docs.qbraid.com/lab/user-guide/getting-started)
```

Comment (beckykd) [reply docs/guides/online-lab-environments.mdx]: They actually don't use the term "Jupyter", so I just used "Lab", which is how they talk about it.

File: docs/guides/online-lab-environments.mdx [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+
+In addition to downloading and installing Qiskit to work on your local machine, there are several options available for those who prefer to work in an online development environment.
+
+Several providers offer online Jupyter environments with Qiskit preinstalled for fast and convenient onboarding. These recommended providers include the cloud service provider OVHcloud and the quantum software development cloud platform qBraid.
+
+Additional online Jupyter notebook environments like IBM Watson&reg; Studio, Google Colab, and Microsoft Azure Machine Learning Studio can be used to work with Qiskit by following the instructions below.
+
+## Recommended notebook environments with Qiskit preinstalled
+
+### OVH Cloud AI notebooks
```
Comment (beckykd) [inline docs/guides/online-lab-environments.mdx:17]:
```suggestion
### OVHcloud AI notebooks
```

**Pull request merged**